core/root/cluster.go

```diff
@@ -186,13 +186,17 @@ func (c clusterImpl) Create() error {
 		return err
 	}
 
+	q := make(chan struct{}, 1)
+	defer func() { q <- struct{}{} }()
+
 	if c.controlPlane.CloudWatchLogging.Enabled && c.controlPlane.CloudWatchLogging.LocalStreaming.Enabled {
-		// Return Journald logs in a separate GoRoutine
-		q := make(chan struct{})
-		defer func() { q <- struct{}{} }()
 		go streamJournaldLogs(c, q)
 	}
 
+	if c.controlPlane.CloudFormationStreaming {
+		go streamCloudFormation(c, cfSvc, q)
```

Comment:
Nit: I'd prefer `streamStackEvents` so that it would better be consistent with `stackProvisioner` immediately after this line, while better representing it streams not general info of a cfn stack but just events.